### DISCUSSION TITLE
Anyone got DeepSeek gguf working?

### DISCUSSION THREAD

**Discussion by ye7iaserag** [oobabooga/text-generation-webui#4495] [General]
Trying Q8 and getting `ERROR: byte not found in vocab: ''` upated llama-cpp-python with no luck

**Comment by ahoepf** [comment]
same problem
with llama.cpp on Nvidia Tesla p40
Linux

**Reply by ye7iaserag** [reply]
It worked for me but with `--cpu`

**Comment by hlk-trkn** [comment]
```
ERROR: byte not found in vocab: '
'
```
with M1Max 32GB

**Reply by ye7iaserag** [reply]
Still with latest llama-cpp-python, only cpu works

**Comment by ruizcrp** [comment]
having the same issue
